forge16/src/main/resources/META-INF/mods.toml

```diff
@@ -5,7 +5,7 @@ issueTrackerURL="https://github.com/EnvyWare/BetterDexRewards/issues"
 
 [[mods]]
 modId="pokeremoval"
-version="2.0.2"
+version="2.0.3"
```

Comment:
You shouldn't bump the version in this MR

Reply:
Oh sorry will change it back, but out of curiosity is that done automatically every time you make a release?